src/cli/README_PERF.md

```diff
@@ -0,0 +1,480 @@
+# OpenThread CLI - Perf Example
+
+## Overview
+
+Cli Perf is a performance tool used to test throughput, latency, jitter, out of order and loss rate of OpenThread Network.
+
+## Quick Start
+
+### Build with Time Sync and Cli Perf support
+
+Use the `TIME_SYNC=1` build switch to enable Time Sync API support.
+Use the `CLI_PERF=1` build switch to enable Cli Perf support.
+
+Note: Latency measurement is based on Time Sync feature.
+
+```bash
+> ./bootstrap
+> make -f examples/Makefile-nrf52840 TIME_SYNC=1 CLI_PERF=1 
+```
+
+### Example 1: Test Unicast Traffic
+
+#### Form Network and enable Time Sync
+
+Form a network with at least two devices. All devices are time synchronized with each other.
+
+#### Node 1
+
+On node 1, create a listening session and start it.
+
+```bash
+> perf server
+Done
+> perf start
+Server listening on UDP port 5001
+Done
+> 
+```
+
+#### Node 2
+
+On node 2, create a transmission session and start it.
+
+```bash
+> perf client destaddr fdde:ad00:beef:0:0:ff:fe00:fc00 bandwidth 2000 length 64
+Done
+> perf start
+Client connecting to  fdde:ad00:beef:0:0:ff:fe00:fc00 , UDP port 5001
+[ ID]  Interval              Transfer     Bandwidth
+Done
+> [  0]  0.000 -  1.280 sec     320 Bytes    2000 bits/sec  
+[  0]  1.280 -  2.304 sec     256 Bytes    2000 bits/sec  
+[  0]  2.304 -  3.328 sec     256 Bytes    2000 bits/sec  
+[  0]  3.328 -  4.352 sec     256 Bytes    2000 bits/sec  
+[  0]  4.352 -  5.376 sec     256 Bytes    2000 bits/sec  
+[  0]  5.376 -  6.400 sec     256 Bytes    2000 bits/sec  
+[  0]  6.400 -  7.424 sec     256 Bytes    2000 bits/sec  
+[  0]  7.424 -  8.448 sec     256 Bytes    2000 bits/sec  
+[  0]  8.448 -  9.472 sec     256 Bytes    2000 bits/sec  
+[  0]  0.000 - 10.240 sec    2560 Bytes    2000 bits/sec 
+```
+
+#### Node 1
+
+On node 1, it shows the test result.
+
+```bash
+> 
+[ ID] Interval             Transfer     Bandwidth         Jitter    Lost/Total LossRate Latency(min, avg, max)
+[  0] local fdde:ad00:beef:0:0:ff:fe00:fc00 port 5001 connected with fdde:ad00:beef:0:3307:2f7b:7426:af07 port 49156
```

Comment:
Why this line appears here?

Reply:
When the server receives the first data packet of the connection, it will print out the infomation about the connection.